### PR TITLE
feat(app): implements sidebar for Linkedin platform

### PR DESCRIPTION
**Pull request by gdixon**
This PR exposes the Linkedin Provider via the Linkedin Platform sidebar.

--

Closes: #456,  closes: #455, closes: #454

### CODE REVIEW

File: app/components/PlatformCards/LinkedinPlatform.tsx [hunk 1]
```diff
@@ -9,21 +9,31 @@ import { debounce } from "ts-debounce";
...
 import { UserContext } from "../../context/userContext";
-import { ProviderSpec } from "../../config/providers";
 import { CeramicContext } from "../../context/ceramicContext";
 
+// --- Platform definitions
+import { getPlatformSpec } from "../../config/platforms";
+import { STAMP_PROVIDERS } from "../../config/providers";
+import { SideBarContent } from "../SideBarContent";
+
 // Each provider is recognised by its ID
```
Comment (aminah-io) [inline app/components/PlatformCards/LinkedinPlatform.tsx:35]:
```suggestion
// Each platform is recognised by its ID
```

File: app/components/PlatformCards/LinkedinPlatform.tsx [hunk 2]
```diff
@@ -38,17 +48,37 @@ function generateUID(length: number) {
...
   const { address, signer } = useContext(UserContext);
-  const { handleAddStamp, allProvidersState } = useContext(CeramicContext);
+  const { handleAddStamps, allProvidersState } = useContext(CeramicContext);
   const [isLoading, setLoading] = useState(false);
   const [state, setState] = useState("");
+  const [canSubmit, setCanSubmit] = useState(false);
+
+  // find all providerIds
+  const providerIds =
+    STAMP_PROVIDERS["Facebook"]?.reduce((all, stamp) => {
```
Comment (aminah-io) [inline app/components/PlatformCards/LinkedinPlatform.tsx:58]:
```suggestion
    STAMP_PROVIDERS["Linkedin"]?.reduce((all, stamp) => {
```

**Review by aminah-io — CHANGES_REQUESTED** [review]
Just a couple of suggestions, otherwise LGTM 🚀 !

**Review by aminah-io — APPROVED** [review]
LGTM 🚀 !!

File: app/components/PlatformCards/LinkedinPlatform.tsx [hunk 3]
```diff
@@ -109,21 +140,40 @@ export default function LinkedinCard(): JSX.Element {
...
+          setVerifiedProviders([...verifiedProviders]);
+          setSelectedProviders([...verifiedProviders]);
+          // reset can submit state
+          setCanSubmit(false);
           // Custom Success Toast
           toast({
             duration: 5000,
             isClosable: true,
-            render: (result: any) => <DoneToastContent providerId={providerId} result={result} />,
+            render: (result: any) => <DoneToastContent providerId={platformId} result={result} />,
```
Comment (gdixon) [inline app/components/PlatformCards/LinkedinPlatform.tsx:171]:
```suggestion
            render: (result: any) => <DoneToastContent platformId={platformId} result={result} />,
```

**Pull request merged**